src/applications/simple-forms/21-10210/containers/ConfirmationPage.jsx

```diff
@@ -0,0 +1,93 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { format, isValid } from 'date-fns';
+import { connect } from 'react-redux';
+
+import scrollToTop from 'platform/utilities/ui/scrollToTop';
+import { focusElement } from 'platform/utilities/ui';
+
+export class ConfirmationPage extends React.Component {
+  componentDidMount() {
+    focusElement('h2');
+    scrollToTop('topScrollElement');
+  }
+
+  render() {
+    const { form } = this.props;
+    const { submission, formId, data } = form;
+    const submitDate = format(submission.timestamp, 'YYYY-MM-DD');
```

Comment:
This submitDate declaration's just a guess on my part, let me know if it's correct.
I had to add it to fix 2 lint errors in the generated code:
- submission defined but never used; and
- submitDate undefined.